### PR TITLE
Fixed #2983 - translation of fullcalendar fixed

### PR DESCRIPTION
**Pull request by plessmann**
## Description
changed option lang to locale

## Motivation and Context
Solves translation issues with weekdays etc. in Calendar

## How To Test This
Change the language from en_us to de_DE and check this url
https://[SUGARDOMAIN]/index.php?module=Calendar&action=index&parentTab=Alle
The weekdays should be translated to German: Sonntag, Montag, Dienstag, Mittwoch, Donnerstag, Frreitag, Samstag. Also the literal "Ganztätig" should be translated.
The buttons "Shared Month" and "Shared Weeks" are not fixed by this bugfix.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist
- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

**Comment by horus68** [comment]
Its a duplicated PR from https://github.com/salesagility/SuiteCRM/pull/2983
What we need its that PR merged ... since 25 Jan!

File: modules/Calendar/Cal.js [hunk 1]
```diff
@@ -151,11 +151,11 @@ CAL.remove_edit_dialog();var cal_loaded=true;$($.fullCalendar).ready(function(){
...
+$('#calendar'+user_id).fullCalendar({header:{left:'',center:'',right:''},locale:global_langPrefix,views:views,minTime:global_start_time,maxTime:global_end_time,selectHelper:true,selectable:true,selectOverlap:true,slotMinutes:global_timeslots,defaultDate:global_year+"-"+global_month+"-"+global_day,editable:global_edit,disableDragging:global_items_draggable,eventLimit:true,defaultView:global_view,firstDay:global_start_week_day,height:global_basic_min_height,columnFormat:headerFormat,select:function(date,jsEvent,view){if(global_edit==true){var date_start=date.format(global_datetime_format);var date_end=jsEvent.format(global_datetime_format);var date_duration=jsEvent.diff(date);if(date.hasTime()==false){var date_end=date.add(1,'days').format(global_datetime_format);}
```
Comment (daniel-samson) [inline modules/Calendar/Cal.js]: Please also update jssource/src_files/modules/Calendar/Cal.js and include in your pull request. I think it is around line 913

File: modules/Calendar/Cal.js [hunk 2]
```diff
@@ -151,11 +151,11 @@ CAL.remove_edit_dialog();var cal_loaded=true;$($.fullCalendar).ready(function(){
...
\ No newline at end of file
+api.set('content.text',result.body);}).fail(function(){$(divBody).html(SUGAR.language.translate('app_strings','LBL_EMAIL_ERROR_GENERAL_TITLE'));}).always(function(){});}},position:{my:'bottom left',at:'top left'},show:{solo:true},hide:{event:false},style:{width:224,padding:5,color:'black',textAlign:'left',border:{width:1,radius:3},tip:'bottomLeft',classes:{tooltip:'ui-widget',tip:'ui-widget',title:'ui-widget-header',content:'ui-widget-content'}}});}},}).ready(function(){$(window).resize();});if($('#calendar_title_'+user_id).length==0){var calendar=$("#calendar"+user_id+" > .fc-view-container");var calendarTitle="<div class='monthCalBody'><h5 class='calSharedUser' id='calendar_title_"+user_id+"'></h5></div><div id='calendar"+user_id+"'></div>";$(calendarTitle).prependTo(calendar);$.ajax({url:"index.php?module=Calendar&action=getUser&record="+user_id,}).done(function(data){data=jQuery.parseJSON(data);$("#calendar_title_"+user_id).html(data.full_name);});}}});
```
Comment (daniel-samson) [inline modules/Calendar/Cal.js]: Please also update jssource/src_files/modules/Calendar/Cal.js and include in your pull request. I think it is around line 1070

**Comment by Dillon-Brown** [comment]
Duplicate of #2983 where this is already fixed and ready to merge.

**Pull request closed without merging**